lib/tasks_private/scvmm.rake

```diff
@@ -0,0 +1,54 @@
+require 'winrm'
+require 'json'
+require 'yaml'
+
+namespace :manageiq do
+  namespace :providers do
+    namespace :scvmm do
```

Comment:
I put a private task in the aws provider at top level namespace, see [here](https://github.com/ManageIQ/manageiq-providers-amazon/blob/087f2bb574304ce915945425f8e91d0252fc492a/lib/tasks_private/instance_types.rake)

The `manageiq:providers` namespace was useful when the tasks where included also in the core app, but thats no longer the case.

Up to you to leave it namespaced or not